conan/cli/formatters/report/diff_html.py

```diff
@@ -0,0 +1,137 @@
+diff_html = r"""
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>{{ old_reference }} - {{ new_reference }}</title>
+        <style>
+            body { font-family: monospace; margin: 0px; }
+            .container { display: flex; height: 100%; }
+            .sidebar {
+                min-width: 20%;
+                max-width: 20%;
+                padding: 10px;
+                overflow-y: scroll;
+                background: #f4f4f4;
+                border-right: 1px solid #ccc;
+            }
+            .sidebar li { line-height: 1.5; }
+            .content {
+                padding: 20px;
+                background: #fff;
+                overflow-y: scroll;
+            }
+            .content span {
+                white-space: pre-wrap;
+            }
+            .add { background-color: #76ffbb; }
+            .del { background-color: #fdb9c1; }
+            .context, .diff-content { background-color: #f8f8f8; }
+            .filename { background-color: #ffffff; }
+            a:visited {
+                color: blue;
+            }
+        </style>
+        <script>
+            let includeSearchQuery = "";
```

Comment:
@uilianries expressed the need to have an exclusion filter and I agree, it muddies the logic a tiny bit but works good